### PR TITLE
 feat: 🎸 Add signup button

### PR DESCRIPTION
**Pull request by futurechallenger**
1. Add "Sign Up" button.
2. Related switch with this feature.
3. When this button is clicked, it will lead to [RC register page](https://www.ringcentral.com/office/plansandpricing.html)

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -21,11 +21,14 @@
     "docs/**": true
   },
   "files.eol": "\n",
-  // Disable default VSCode formatting for JavaScript
+  // Disable default VSCode formatting for JavaScript	   // Disable default VSCode formatting for JavaScript
```
Comment (embbnux) [inline .vscode/settings.json]: Should not add unrelated codes in feature PR

File: packages/ringcentral-widgets/containers/WelcomePage/index.js [hunk 2]
```diff
@@ -46,6 +50,14 @@ function mapToFunctions(_, {
     onLoginButtonClick() {
       oAuth.openOAuthPage();
     },
+    onSignUpButtonClick() {
+      const url = 'https://www.ringcentral.com/office/plansandpricing.html';
+      if (chrome && chrome.tabs) {
```
Comment (embbnux) [inline packages/ringcentral-widgets/containers/WelcomePage/index.js]: Please don't add platform related codes in common widgets library

Comment (futurechallenger) [reply packages/ringcentral-widgets/containers/WelcomePage/index.js]: done

File: .vscode/settings.json (hunk 1 above)
Comment (futurechallenger) [reply .vscode/settings.json]: done

File: packages/ringcentral-widgets/containers/WelcomePage/index.js (hunk 2 above)
Comment (futurechallenger) [reply packages/ringcentral-widgets/containers/WelcomePage/index.js]: done

**Pull request merged**